### ISSUE TITLE
No Data in Parquet Materialization

### ISSUE THREAD

**Issue by dyaffe** [estuary/connectors#215]
**Problem**
I created two materializations to Parquet which used a seemed to be successful, and no data made it to one but the other succeed fine.

**Background**

When using a collection that contained commas, symbols the reserved keyword "TIMESTAMP" as well as that name, the materialization appeared to succeed but silently failed.  It successfully ran when I removed those characters.

**Other problems**
Even though I named a prefix, my file was not actually under that prefix.

My failed materialization was named: `estuary/dave/s3-parquet`

My successful one was: `estuary/dave/clean-sheet-parquet`

See the second comment below which has a lot more info figured out.

**Comment by dyaffe** [comment]
To add to the complications here, my data is now showing up, but it's going to two  pefixes.  I wrote all data to one prefix "test/".  The second prefix is "sheeeeeet" which is not part of any of my materializations.

Furthermore, it file updates seem to not be happening at the stated interval, but much slower.  I had used 60 seconds for one and 80 seconds for the other.  It appears that they're updating significantly slower?

**Comment by dyaffe** [comment]
So I figured out a bunch of things that are going on -- I had created a parquet materialization that never made it to S3 21 hours earlier.  After creating the 3rd one it appeared to make it to S3.  The one with the "sheeeeeet" prefix was the old one and I had created two with the "test/" prefix.  

All three got un-wedged when I submitted the last materialization and got written together.  What type of reason would behavior like this happen and can we prevent it in the future?

The `uploadIntervalInSeconds` appears to be rounded up to the next 5 minute interval instead of what that value takes.  Presumably that's cause this is how Flow runs.  If that's true, should we change docs to suggest that it should be in increments of 5 minutes?
